### DISCUSSION TITLE
Stress intensity boundary conditions

### DISCUSSION THREAD

**Discussion by sidharthsarmah** [idaholab/moose#23120] [Q&A General] [answered]
Hi,
I am trying to implement a stress intensity BC on the Outer nodes in a Boundary layer Problem (Green line) and a symmetry condition on bottom (Red line), which looks something like this.

It is a coupled problem with concentration BC on the H surface(Blue line)

```
#****************BCs********************

[BCs]

   [./surface]
    type = ADDirichletBC
    variable = conc_lattice


  [./x_bot]
    type = ADDirichletBC
    variable = disp_x
    boundary = 'Symmetry'
    value = 0  
  [../]

  [./y_bot]
    type = ADDirichletBC
    variable = disp_y
    boundary = 'Symmetry'
    value = 0  
  [../]

[]
```
I have defined the Outer nodes in the input files.

```

```
Could you please suggest if there is any mistake as I am encountering a  convergence issue)

**Comment by GiudGiud** [comment, marked as answer]
I think the issue was found on another thread. 
Units are MPa for stress instead of Pa
